### DISCUSSION TITLE
Enhance Java App Appearance with GTK+ Font & Theme Settings · @mrsnailo

### DISCUSSION THREAD

**Discussion by giscus** [mrsnailo/mrsnailo.github.io#105] [General]
# Enhance Java App Appearance with GTK+ Font & Theme Settings · @mrsnailo

Improve the visual appeal of your Java applications by aligning them with GTK theme styles and font settings

https://mrsnailo.github.io/posts/2024/01/enhance-java-app-appearance-with-gtk-font-theme-settings/

**Comment by SharafatKarim** [comment]
Awesome post bro! Thanks 😍
